src/providers/zerion.rs

```diff
@@ -110,11 +110,21 @@ pub struct ZerionTransactionTransferQuantity {
     pub numeric: String,
 }
 
+/// Implementation details of the fungible on various chains.
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub struct ZerionTransactionImplementation {
+    /// Unique id of the chain.
+    pub chain_id: String,
+    /// Implementation address on the chain.
+    pub address: Option<String>,
+}
+
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
 pub struct ZerionTransactionFungibleInfo {
     pub name: Option<String>,
     pub symbol: Option<String>,
     pub icon: Option<ZerionTransactionURLItem>,
+    pub implementations: Vec<ZerionTransactionImplementation>,
```

Comment:
```suggestion
    pub chain_id: String,
```

Tenet of this service is to be provider agnostic. So let's refrain from exposing as much data as possible from Zerion's API and instead expose only the required data such that if we ever want to add another provider it becomes easier.